src/ProjectTemplates/Web.ProjectTemplates/content/Grpc-Application/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "gRPC Application",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1027"
+  },
+  "description": {
+    "text": "A project template for creating a gRPC ASP.NET Core application.",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1028"
+  },
+  "order": 100,
+  "icon": "vs-2017.3/Empty.png",
```

Comment:
Should this use [Empty](https://github.com/aspnet/AspNetCore/blob/d1fac577347976657b5dc8ba0d1fe6462fcf7b85/src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/.template.config/vs-2017.3/Empty.png) or [Web](https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/.template.config/vs-2017.3/WebApplication.png)